backend/codesirius/internal_api/views/hidden_test_bundle.py

```
@@ -0,0 +1,51 @@
+            return Problem.objects.get(id=problem_pk)
+        except Problem.DoesNotExist:
+            raise NotFound("Problem not found")
+        except Exception as e:
+            logger.exception(e)
+            raise NotFound("Failed to retrieve problem")
+
+    def post(self, request: Request, problem_pk: int) -> CodesiriusAPIResponse:
+        problem = self.get_problem(problem_pk)
+        logger.info(f"Creating hidden test bundle for problem {problem.id}")
+        if hasattr(problem, "hidden_test_bundle"):
+            raise ValidationError("Hidden test bundle already exists for this problem")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)